backend/yugioh/management/commands/seed.py

```diff
@@ -0,0 +1,240 @@
+                    self.stdout.write(progress_message, ending='\r')
+                    self.stdout.flush()
+
+                    card_set_name = entry["set_name"]
+                    card_set_code = entry["set_code"]
+                    rarity = entry["set_rarity"]
+                    rarity_code = entry["set_rarity_code"]
+
+                    # if rarity_code == '':
+                    #     code_generation = rarity.split()
+                    #     for word in code_generation:
+                    #         rarity_code += word[0]
+                    # print(rarity_code)
+
+                    try:
+                        rarity_object = YugiohCardRarity.objects.get(rarity=rarity, rarity_code=rarity_code)
+                    except YugiohCardRarity.DoesNotExist:
+                        rarity_object = None
+
+                    try:
+                        card_set_object = YugiohCardSet.objects.filter(card_set_name=card_set_name,
+                                                                       set_code=card_set_code).get()
+                    except YugiohCardSet.DoesNotExist:
+                        card_set_object = None
+
+                    if not rarity_object:
+                        #                     self.stdout.write('Rarity does not exist... Creating')
+                        logging.info('Rarity does not exist... Creating')
+                        rarity_object = YugiohCardRarity.objects.create(rarity=rarity, rarity_code=rarity_code)
+
+                    if not card_set_object:
+                        #                     self.stdout.write('Cardset does not exist... Creating')
+                        logging.info('Cardset does not exist... Creating')
+                        card_set_object = YugiohCardSet.objects.create(card_set_name=card_set_name,
+                                                                       set_code=card_set_code)
+
+                    existing_card = YugiohCardInSet.objects.filter(
+                        rarity=rarity_object,
+                        set=card_set_object,
+                        yugioh_card=yugioh_card_object
+                    ).first()
+
+                    if not existing_card:
+                        YugiohCardInSet.objects.create(rarity=rarity_object, set=card_set_object,
+                                                       yugioh_card=yugioh_card_object)
+                    else:
+                        # self.stdout.write(self.style.ERROR(f'Card {yugioh_card_object} '
+                        #                                    f'{card_set_object} {rarity_object}... Skipping'))
+                        logging.info(f'Card {yugioh_card_object} with '
+                                     f'{card_set_object} {rarity_object}... Skipping')
+            except KeyError:
+                logging.info(f'No sets for {card_name}')
+
+        # self.stdout.write(self.style.SUCCESS(f'DONE!'))
+        logging.info('DONE!')
```

Comment:
The end print is usually good to provide the user with a super compact summary of what happened, e.g., how many objects were created, etc.